### DISCUSSION TITLE
secureLocalStorage jest unit test in reactjs

### DISCUSSION THREAD

**Discussion by tantran95** [sushinpv/react-secure-storage#34] [Q&A] [answered]
Hi, have an issue with secureLocalStorage in jest unit test. 
<img width="739" alt="Screenshot 2023-09-07 at 10 21 24" src="https://github.com/sushinpv/react-secure-storage/assets/135570757/8bf85f6d-69eb-4a90-9ca3-ce4213fb5c42">
Do you have any solution to resolve this one. Thanks

**Comment by sushinpv** [comment, marked as answer]
Hi @tantran95 Please have a look at the solution discussed here https://github.com/sushinpv/react-secure-storage/issues/12

**Reply by tantran95** [reply]
Thanks @sushinpv. The issue is resolved.
